src/components/checkbox/CdrCheckbox.vue

```diff
@@ -13,19 +13,24 @@
         :class="[style['cdr-checkbox__input'], inputClass]"
         type="checkbox"
         v-bind="$attrs"
-        :checked="newValue"
-        @change="$emit('update:modelValue', $event.target.checked, $event)"
         :true-value="customValue ? null : trueValue"
         :false-value="customValue ? null : falseValue"
         :value="customValue"
-        :indeterminate="indeterminate"
-      >
+        v-indeterminate="indeterminate"
```

Comment:
Not sure I like the use of `v-` here. To me this looks like like a vue-managed attribute and could be confusing ... also potentially breaking if they ever decided to use that themselves. Why not `isIndeterminate` or something like that?

Reply:
Disregard, I learned something new today!